model/setting.go

```diff
@@ -4,10 +4,11 @@ import "time"
 
 // UserSetting a setting for a user.
```

Comment:
Sorry, overlooked this PR. I'm okay with merging this but I've one question: Do you think it would be cool to stack the native date time inputs like this? This makes the TimeSpan selection slightly bigger but allows for more area for the tag inputs.
![image](https://github.com/user-attachments/assets/713b8eea-e88e-4506-8fee-b6ee1523d889)

Reply:
Aesthetically, I think mixing one-line and two-line content blocks on the same list item looks a little strange. Functionally, more space for the tags is nice.

I would opt to leave it as-is, but if there's strong interest in stacking the inputs, I'll put in another PR to do so.

Reply:
:+1: